### ISSUE TITLE
[Suggestion] Viewing images that are not sent using a link

### ISSUE THREAD

**Issue by VilotStar** [SebOuellette/LiveBot#59]
I think it would be cool if the bot could get images from servers and display them.

**Comment by dcrime** [comment]
First I'd like to say, please use [our discord server for any suggestions](https://discord.gg/TMUc8Bs)

For now until embeds are done we aren't thinking about doing anything about attachments, link previews are treated as embeds so we still have to finish that first, there's no ETA because of time issues and a low developer team (2 members), first we'd need to complete the video controls instead of using the default ones because the goal of this is to make it look like discord, also there are around 6 types of embeds if I'm not wrong and they each follow their own rules

So in short, it can be done but there are other stuff being prioritized at the moment because we want it to work as good as it can and to look as good as it can
